torchvision/models/googlenet.py

```diff
@@ -60,37 +60,115 @@ def googlenet(pretrained=False, progress=True, **kwargs):
     return GoogLeNet(**kwargs)
 
 
+class BasicConv2d(nn.Module):
+
+    def __init__(self, in_channels, out_channels, **kwargs):
+        super(BasicConv2d, self).__init__()
+        self.conv = nn.Conv2d(in_channels, out_channels, bias=False, **kwargs)
+        self.bn = nn.BatchNorm2d(out_channels, eps=0.001)
+
+    def forward(self, x):
+        x = self.conv(x)
+        x = self.bn(x)
+        return F.relu(x, inplace=True)
+
+
+class Inception(nn.Module):
```

Comment:
Is this code movement necessary? This makes it harder to review.

Reply:
Hi @fmassa , this is because these class names such as Inception are passed into init() of GoogLeNet class, and if don't move them above GoogLeNet class, it will generate error such as:
```
NameError: name 'InceptionAux' is not defined
```

This movement is only needed for googlenet and inception, since other models have main module class at the bottom.